include/bluetooth/direction.h

```diff
@@ -217,4 +230,22 @@ int bt_df_conn_cte_rx_enable(struct bt_conn *conn, const struct bt_df_conn_cte_r
  */
 int bt_df_conn_cte_rx_disable(struct bt_conn *conn);
 
+/**
+ * @brief Set Constant Tone Extension transmission parameters for a connection.
+ *
+ * The function is available if @kconfig{CONFIG_BT_DF_CONNECTION_CTE_TX} is enabled.
+ *
+ * @note @p params contents may not be completely validated and forwarded to Controller.
+ * If bt_df_conn_cte_tx_param.cte_types bits BT_DF_CTE_TYPE_AOD_1US and BT_DF_CTE_TYPE_AOD_2US are
+ * not set, the function will not validate members bt_df_conn_cte_tx_param.num_ant_ids and
+ * bt_df_conn_cte_tx_param.ant_ids. Also these members of @p params will not be forwarded to
+ * Controller.
```

Comment:
I'm also trying to understand what is being said here. 

So if `bt_df_conn_cte_tx_param.cte_types` does not set both `BT_DF_CTE_TYPE_AOD_1US` and `BT_DF_CTE_TYPE_AOD_2US`, then the antenna paramaters are not validated, or sent to the controller? 

If that is correctly understood, perhaps a different phrasing such as 

`if both `BT_DF_CTE_TYPE_AOD_1US` and `BT_DF_CTE_TYPE_AOD_2US` are not set in the bitfields, the `bt_df_conn_cte_tx_param.num_ant_ids` and `bt_df_conn_cte_tx_param.ant_ids` parameters will be ignored`. 

Alternatively, we could return an error, and add explicit requirement in the struct information that those values are only settable (or valid) if `cte_types` has both bits set.

Reply:
You have interpreted it correctly and thank you I'll update description according to suggestion.

The note is just to inform end user. In conditions described in the note, the parameters are irrelevant. Controller will not use them (this is according to BT 5.3 Core spec), hence there is no need to return error.

Reply:
That is fine :) (I can't remember what we usually do if the application provides parameters that are ignored by the controller). 

Is it also correct understood that both bits need to be set, or just one of them?

Reply:
Just one of these bits need to be set to execute complete parameters validation and forward them to controller.

Reply:
Okay, as I read it, that's not clear from what the current text says either: 

> If bt_df_conn_cte_tx_param.cte_types bits BT_DF_CTE_TYPE_AOD_1US and BT_DF_CTE_TYPE_AOD_2US are not set

Could be interpreted as both need to be set.

Reply:
Ok, thank you for your advice.
I've used form proposed by you. 
Have a look please.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/41466#discussion_r776773612